### ISSUE TITLE
Birdeye default camera

### ISSUE THREAD

**Issue by diegargon** [blakeblackshear/frigate#14031]
Hi

Especially in birdeye full screen mode i would like to have a option to set a default camera

The idea is to put the camera at the entrance to my property that only activates alerts when there is someone near the entrance but not on the road. It would be useful if if there is no movement on any camera by default it would show that camera instead of being dark.

**Comment by NickM-27** [comment]
Just make that camera continuous so it always shows

**Comment by DrSpaldo** [comment]
@diegargon this is what I have done for my birdseye. I have the front entry as continuous as @NickM-27 suggested. 

```
birdseye:
  enabled: true
  restream: true
  width: 2592
  height: 1944
  quality: 1
  layout:
    max_cameras: 1
```

Then I have turned on cameras individually (per camera settings) like this example;

```
    birdseye:
      order: 5
      mode: objects
```
I've made them ordered in my own order of importance. It works quite good for the most part. _There are a few little tweaks to make it perfect that hopefully will eventually make the feature list (person only detection & less switching between cameras)_
